core/src/org/pentaho/di/core/row/value/ValueMetaBase.java

```diff
@@ -578,6 +588,70 @@ public void setCaseInsensitive( boolean caseInsensitive ) {
     this.caseInsensitive = caseInsensitive;
   }
 
+   /**
+   * @return the collatorDisabled
+   */
+  @Override
+  public boolean isCollatorDisabled() {
+    return collatorDisabled;
+  }
+
+  /**
+   * @param collatorDisabled
+   *          the collatorDisabled to set
+   */
+  @Override
+  public void setCollatorDisabled( boolean collatorDisabled ) {
+    this.collatorDisabled = collatorDisabled;
+  }
+  
+  
+    /**
+   * @ sets the collator Locale
+   */
+  @Override
+  public void setCollatorLocale( Locale locale ) {
+    this.collator = Collator.getInstance( locale );
+  }
+
+    /**
+   * @get the collatorStrength
+   */
+  @Override
+  public String getCollatorStrength() {
+    return collatorStrength;
+  }
+
+  /**
+   * @param collatorStrength
+   *          the collatorStrength to set
+   */
+  @Override
+  public void setCollatorStrength( String collatorStrength ) {
+    switch ( collatorStrength.toUpperCase() ) {
+      case "PRIMARY":
+        this.collatorStrength = collatorStrength.toUpperCase();
+        this.collator.setStrength( 0 );
+        break;
+      case "SECONDARY":
+        this.collatorStrength = collatorStrength.toUpperCase();
+        this.collator.setStrength( 1 );
+        break;
+      case "TERTIARY":
+        this.collatorStrength = collatorStrength.toUpperCase();
+        this.collator.setStrength( 2 );
+        break;
+      case "IDENTICAL":
+        this.collatorStrength = collatorStrength.toUpperCase();
+        this.collator.setStrength( 3 );
+        break;
+      default:
```

Comment:
Disagree here - if they don't pass in a valid collator, it should throw an exception here.